config-env.sh

```diff
@@ -0,0 +1,11 @@
+# Configure this environment variable to be the path to the downloaded SDK
```

Comment:
This file shouldn't be checked in; I suggest adding it to `.git/info/exclude` or whatever the local gitignore file is.

Reply:
Removed